apps/api-documenter/src/markdown/MarkdownEmitter.ts

```diff
@@ -106,23 +104,9 @@ export class MarkdownEmitter {
       }
       case DocNodeKind.CodeSpan: {
         const docCodeSpan: DocCodeSpan = docNode as DocCodeSpan;
-        if (context.insideTable) {
-          writer.write('<code>');
-        } else {
-          writer.write('`');
-        }
-        if (context.insideTable) {
-          const code: string = this.getTableEscapedText(docCodeSpan.code);
-          const parts: string[] = code.split(/\r?\n/g);
-          writer.write(parts.join('</code><br/><code>'));
-        } else {
-          writer.write(docCodeSpan.code);
-        }
-        if (context.insideTable) {
-          writer.write('</code>');
-        } else {
-          writer.write('`');
-        }
```

Comment:
All of this extra handling code is not necessary anymore, since we can just emit markdown inside the HTML tables.